smart_card_connector_app/src/window-devices-displaying.js

```diff
@@ -85,21 +92,21 @@ function displayReaderList(readers) {
     goog.asserts.assert(readersListElement);
 
     var indicatorClasses = 'reader-state-indicator reader-state-indicator-' +
-                           reader['status'];
+                            reader['status'];
     if (reader['status'] == GSC.PcscLiteServer.ReaderStatus.SUCCESS &&
         reader['isCardPresent']) {
       indicatorClasses = 'reader-card-present-indicator';
     }
     var indicator = goog.dom.createDom('span', indicatorClasses);
 
     var indicatorContainer = goog.dom.createDom(
-        'span', 'reader-indicator-container', indicator);
+      'span', 'reader-indicator-container', indicator);
```

Comment:
nit: Ditto for unwanted indentation change.

Same in line 106.

See https://google.github.io/styleguide/jsguide.html#formatting-indent